tests/examples/source/devfiles/springboot/devfile.yaml

```diff
@@ -5,47 +5,37 @@ metadata:
 starterProjects:
   - name: springbootproject
     git:
-      location: "https://github.com/maysunfaisal/springboot.git"
+      location: "https://github.com/odo-devfiles/springboot-ex.git"
 components:
   - container:
       name: tools
-      image: maysunfaisal/springbootbuild
+      image: quay.io/eclipse/che-java11-maven:nightly
       memoryLimit: 768Mi
       command: ['tail']
       args: [ '-f', '/dev/null']
-      volumeMounts:
-        - name: springbootpvc
-          path: /data
       mountSources: true
   - container:
       name: runtime
-      image: maysunfaisal/springbootruntime
+      image: quay.io/eclipse/che-java11-maven:nightly
       memoryLimit: 768Mi
-      command: ['tail']
-      args: [ '-f', '/dev/null']
       endpoints:
         - name: "8080/tcp"
           targetPort: 8080
-      volumeMounts:
-        - name: springbootpvc
-          path: /data
-      mountSources: false
-  - volume:
-      name: springbootpvc
+      mountSources: true
 commands:
   - exec:
       id: defaultBuild
       component: tools
-      commandLine: "/artifacts/bin/build-container-full.sh"
+      commandLine: mvn install
```

Comment:
are you planning on having these commands or keeping ti similar to your devfile PR?

Reply:
kept same a registry